dev/com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6/bnd.overrides

```diff
@@ -14,7 +14,8 @@ exportVer=2.6.2
 
 Bundle-Name: Apache CXF Runtime SOAP Binding
 Bundle-Description: Apache CXF Runtime SOAP Binding, version 2.6.2
-Bundle-SymbolicName: com.ibm.ws.org.apache.cxf-rt-bindings-soap.2.6.2
+Bundle-SymbolicName: com.ibm.ws.org.apache.cxf.cxf.rt.bindings.soap.2.6
+Bundle-Activator: com.ibm.ws.jaxws.cxf.rt.bindings.soap.NoOpActivator
```

Comment:
I think it would be better verbage to remove the jaxws and just have com.ibm.ws.cxf.rt.bindings.soap.NoOpActivator